fs-album-slideshow.html

```diff
@@ -1160,6 +1160,11 @@ <h4>[[i18n('slideshow_controls_slideshow_speed')]]</h4>
         player.pause();
         if (resetTime) player.currentTime = 0;
       }
+    },
+    handleGalleryViewChanged: function(e) {
+      // If the gallery view is being set to its initial state, leave the slideshow open
+      if(!e.detail.oldValue) return;
+      this.closeSlideshowOptions();
```

Comment:
Not required, but I would suggest writing it this way because in my experience it's a best practice to use explicit `return` statements sparingly.  If I remember right, I think the reasoning is that it makes the code flow easier to follow.
```suggestion
      if (e.detail.oldValue) {
        this.closeSlideshowOptions();
      }
```